core/build.gradle.kts

```diff
@@ -57,6 +54,11 @@ dependencies {
     api("org.checkerframework:checker-qual")
     api("org.slf4j:slf4j-api")
 
+    api("org.locationtech.jts:jts-core")
+    api("org.locationtech.jts.io:jts-io-common")
+    compileOnly("org.locationtech.proj4j:proj4j")
+    testImplementation("org.locationtech.proj4j:proj4j")
```

Comment:
`testImplementation` means that we need it in the compile and runtime classpath for tests; is this true? If we can use `testRuntimeOnly` I think it would be better.

Reply:
Moreover, consider moving it closer to other dependencies in the same group.